test/functional/api/cas/cli_messages.py

```diff
@@ -15,6 +15,14 @@
     r"Successfully added cache instance \d+"
 ]
 
+start_wo_load_on_cache_dev_prev_used = [
+    r"Error inserting cache \d+",
+    r"Old metadata found on device\.",
+    r"Please load cache metadata using --load option or use --force to",
+    r" discard on-disk metadata and start fresh cache instance\.",
```

Comment:
If this is one line in output, make it one line

Reply:
Actually it is not. This line is broken like that.